src/azul/uuids.py

```diff
@@ -109,6 +109,28 @@ def validate_uuid_prefix(uuid_prefix: str) -> None:
         raise InvalidUUIDPrefixError(uuid_prefix)
 
 
+def change_version(uuid: str, old_version: int, new_version: int) -> str:
```

Comment:
If the current version of the UUID can be pulled out of the UUID itself, why is the `old_version` parameter needed?

Reply:
For the assertion on line 126.